pkg/client.go

```diff
@@ -60,7 +60,7 @@ func handleApiError(resp *http.Response) (returnResponse bool, err error) {
 		return false, err
 	}
 
-	return true, nil
+	return
```

Comment:
Since there are "non-naked" error returns in lines 57 and 60 I would suggest returning `false, err` here as well to remain consistent in this method.